src/fixtures/multi-monorepo.ts

```diff
@@ -0,0 +1,107 @@
+import * as process from 'process';
```

Comment:
Nit: is it common to use kebab-case for ts files? :) The rest of the project files are using camel case.

Reply:
Maybe I was just hungry?! 😛 Thanks for pointing it out!